tapgarden/interface.go

```diff
@@ -59,43 +59,43 @@ type Planter interface {
 }
 
 // BatchState an enum that represents the various stages of a minting batch.
-type BatchState uint8
+type BatchState uint32
```

Comment:
Do we need to update this? I don't think we'll ever have more than 255 states. And since we need to cast for the atomic library anyway, `uint8` should be fine.